### ISSUE TITLE
Update the project creation page to support different experience for deep links

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/connect-app#2047]
Due to the shift of our marketing site pointing directly to project creation forms, we must update the pages to match the experience better. We should be able to replace the back button with "View All Project" (or similar), and remove or redirect the "Close" button.

@vikasrohit - Is it possible to modify the pages to behave differently based on URL parameters for the navigation?
![screen shot 2018-06-13 at 4 58 17 pm](https://user-images.githubusercontent.com/8484061/41384568-72d44bd4-6f2b-11e8-86af-a754bcdeb625.png)

**Comment by vikasrohit** [comment]
Yes, @vic-topcoder it is possible but we have to pass additional query params in URL from marketing sites (e.g. deepLink=true) because we can't differentiate if user is coming directly via deep link or navigated to the step following the wizard.

I think Esc button  honours the `returnUrl` parameter in query string already i.e. if with deep link we provide the `returnUrl` it should take user to that particular page if user cancel the wizard.
